human-biomolecular-data/data-sources.md

```diff
@@ -19,93 +25,134 @@ training:
+By providing this overview of the data sources platforms of infectious diseases using human biomolecular data, this document aims to facilitate the development of new research initiatives and collaborations in the field.
 
-<!--- In the context of BY-COVID, a data source can a repository which should include at least the metadata and ideally the data, that might not be directly available when considering sensitive data. Also, repositories should have the capacity to share this data and therefore have a governance model in place on how to do it. It can also include registries of data sources important for the field, with a direct link to the original data sources to be able to request access to the data. --->
 
-## Introduction
+### What is human biomolecular data, and why is it important for infectious diseases research?
 
-## What is human biomolecular data, and why is it important for infectious diseases research?
+Human biomolecular data refers to information obtained from the analysis of biological molecules, such as DNA, RNA, proteins, and metabolites. This type of data is used to study the molecular mechanisms underlying disease and to identify potential drug targets.
 
-## Topic 1 
+Human clinical data, on the other hand, refers to information obtained from the study of patients, including their medical histories, physical exams, and laboratory tests. This type of data is used to diagnose and treat disease, as well as to evaluate the safety and efficacy of new therapies.
 
-<!--- Subsection related to a specific topic related to the data sources of the category that you selected.--->
+While both types of data are important for understanding human health and disease, they are collected and analysed in different ways and for different purposes.
 
-Short explanation of what this topic is about and why it is important, with an emphasis on infectious diseases and the category that you selected.
+Human biomolecular data is of great importance in infectious disease research because it plays a critical role in understanding the molecular basis of the disease. By analyzing the biomolecular data, researchers can gain a deeper understanding of the disease's pathogenesis, evolution, and transmission. Personalised medicine is another area where biomolecular data can be applied. By analyzing an individual's biomolecular data, researchers can develop personalised treatment plans that are tailored to the specific needs of the patient. For example, if a patient has a genetic mutation that predisposes them to a particular disease, this information can be used to develop a personalised treatment plan that takes into account the patient's genetic profile.
```

Comment:
I think the paragraph is okay, but it feels as if the previously made distinction between Human biomolecular and clinical data was emphasized here, whereas the benefits and uses (e.g. when talking about diseases) require both, and not just the biomolecular data.

Reply:
this change has been addressed